asv_bench/benchmarks/io/sas.py

```diff
@@ -1,30 +1,23 @@
-import os
+from pathlib import Path
 
 from pandas import read_sas
 
+ROOT = Path(__file__).parents[3] / "pandas" / "tests" / "io" / "sas" / "data"
```

Comment:
IIUC this is a style choice orthogonal to the rest of the PR?  no real problem with it, but in general best to minimize these to make it easier to focus on the important bits

Reply:
The ASV file would’ve been very confusing if I left the old code be cause my additions can’t use the old code and then we’d end up with two identical versions.